pkg/storage/util/util.go

```diff
@@ -61,49 +58,6 @@ func UpdateCondition(cr *imageregistryv1.Config, conditionType string, status op
 	cr.Status.Conditions = conditions
 }
 
-// GetInfrastructure gets information about the cloud platform that the cluster is
-// installed on including the Type, Region, and other platform specific information.
-// Currently the install config is used as a backup to be compatible with upgrades
-// from 4.1 -> 4.2 when platformStatus did not exist, but should be able to be removed
-// in the future.
```

Comment:
the future is now!

Reply:
The future isn't quite yet (blocked on [this][1] internal ticket).  Currently you can still get into trouble on 4.3+ with clusters that installed on 4.1 and have updated 4.1 -> 4.2 -> 4.3, because there's no migration code to consume the install-config and populate `PlatformStatus` (e.g. [here][2]).  Can we revert this, and open up some different fix to bug 1776665?  Or am I missing something?

[1]: https://issues.redhat.com/browse/CORS-1173
[2]: https://bugzilla.redhat.com/show_bug.cgi?id=1813343

Reply:
Yes, we can revert this and apply the same fix that we did for 4.2 that just filters the events.

Reply:
The fix from #477 would avoid the issue with clusters which were originally 4.1.